aiidalab_eln/cheminfo/__init__.py

```diff
@@ -37,7 +40,30 @@ def __init__(self, **kwargs):
         self.button_clicked = True
         request_token_button = ipw.Button(description="Request token")
         request_token_button.on_click(self.request_token)
-        self.output = ipw.Output() 
+        
+        self.sample_uuid_widget = ipw.Text(
+            description="Sample ID:",
+            value="",
+            style={"description_width": "initial"},
+        )
+        traitlets.link((self, "sample_uuid"), (self.sample_uuid_widget, "value"))
+        
+        self.file_name_widget = ipw.Text(
+            description="File name:",
+            value="",
+            style={"description_width": "initial"},
+        )
+        traitlets.link((self, "file_name"), (self.file_name_widget, "value"))
+
+        self.spectrum_type_widget = ipw.Text(
+            description="Spectrum type:",
```

Comment:
I know this naming is due to cheminfopy but this (`spectrum`) is maybe not the best choice I made as we also deal with stuff that are not spectra? Due you have better ideas? Something like `technique` might work?

Reply:
or "data type"?

Reply:
seems good to me, will also do the change in cheminfopy